### ISSUE TITLE
[Feature/Addition] Merge Contacts function for EveryAction (VAN)

### ISSUE THREAD

**Issue by mkwoods927** [move-coop/parsons#772]
## Detailed Description
The VAN API has a function to merge two contact records in EveryAction. This issue is for adding a method to the VAN class in Parsons to do this!

## Context
Until recently folks who want to de-dupe their EveryAction databases had only two options: to do so manually for every duplicate pair of records or to pay EveryAction. About 8 months ago VAN added this functionality to their API, and adding it to Parsons would save Parsons users a lot of time and money!

## Possible Implementation
[Here is the VAN API documentation](https://docs.ngpvan.com/reference/peoplevanidmergeinto) 
I'm requesting that Parsons API keys have access to this method.

## Priority
Low

**Comment by shaunagm** [comment]
I've given this the 'blocked' label as, afaik, so long as none of us have access to this method we can't test any code we want to add to Parsons. If/when someone gets access, let me know and I can remove the blocked label.

**Comment by mkwoods927** [comment]
@shaunagm TMC has access! I have a function written up that I've tested against the TMC EveryAction that I was planning to drop here. I'll need support writing a unit test for it!

**Comment by shaunagm** [comment]
Oh great! If you want to pair on writing that unit test, feel free to grab time on my Calendly.

**Comment by mkwoods927** [comment]
Draft function:

```
def merge_contacts(self, primary_vanid, source_vanid):
    """
    Merges two contacts in EveryAction. The source contact record will be deleted as part of the merge
    and its data will be moved into the primary contact record. In cases where fields conflict between
    the two contacts and we can't keep both values, such as if the contacts have different first names,
    the primary contact record's data will be retained.
    
    `Args:`
        primary_vanid: str
            The VANID of the primary contact record.
        source_vanid: str
            The VANID of the source contact record.
    `Returns:`
        The VANID of the primary contact record.
    """
    
    url = f'people/{source_vanid}/mergeInto'
    json = {"vanId": primary_vanid}
    
    r = self.connection.put_request(url, json=json)
    return r
```

**Comment by shaunagm** [comment]
Fixed by #776
